_inc/search/components/api.js

```diff
@@ -0,0 +1,14 @@
+class JetpackSearchAPI {
+	getAPIUrl( query ) {
+		let url = 'https://public-api.wordpress.com/rest/v1.3/sites/20115252/search?q=';
+		url += encodeURIComponent( query );
+		url += '&fields=author';
+		return url;
```

Comment:
You could also use `addQueryArgs` from `@wordpress/url` but I understand if you wouldn't want to add dependencies to frontend. :-)